main.js

```diff
@@ -1 +1,145 @@
-  
+ /*
```

Comment:
All functions are described on global namespace, they should be inside a closure (just a comment from the perspective of JS best code practice's)